### DISCUSSION TITLE
I want to try out duet notebooks on MNIST dataset using PySyft

### DISCUSSION THREAD

**Discussion by dnabanita7** [OpenMined/courses#255] [General]
In the examples section of PySyft, I found a duet notebook tutorial of MNIST. I was thinking of trying to work it out myself by running the owner notebook on my os and other data scientist notebook on VM. Is that possible or is there any other method to do so?

**Comment by leriomaggio** [comment]
Hello @dnabanita7 
    yes, that should do the trick! 

As soon as you're able to establish a connection between the two peers, it should be OK.
 
So, as an initial setup checking, I would be sure that you can `ping` the host from the guest (and viceversa). 
If not, there should be options in your VM manager to allow that. This should help you saving lots of headaches debugging things :)

HTH
